### PR TITLE
Release alpha 15

### PR DESCRIPTION
**Pull request by XVincentX**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,18 @@
...
+
+
+### Bug Fixes
+
+* Prism's build process received some tweaks, but there's more work to do #352
+
+### Features
+
+* Prism now has got a static example fallback in case the `dynamic` flag is not enabled #347
+* The PR Template for the repository has been reduced by half #353
```
Comment (philsturgeon) [inline CHANGELOG.md]: This should be an example of a chore, its not a feature, and a user certainly won't care. :)

**Comment by chris-miaskowski** [comment]
You mind if I add my changes to this branch? @XVincentX

**Comment by XVincentX** [comment]
@chris-miaskowski Better not, create a separate PR for that.

**Comment by chris-miaskowski** [comment]
@XVincentX hmm you realize that my changes are already on master right? So it's just a matter of updating the changelog.md (the release will include my changes anyway, all I'm saying)

**Comment by XVincentX** [comment]
Does not really matter, we're going to need a tag anyway to trigger a release; it's up to you, no harm if you push it here.

**Comment by XVincentX** [comment]
Aaah I see what you mean. Very good point, go ahead! Push it here

**Comment by chris-miaskowski** [comment]
@XVincentX added entry to changelog. lmk if is OK and will merge.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,22 @@
...
+### Bug Fixes
+
+* Prism's build process received some tweaks, but there's more work to do #352
+
+### Features
+
+* Prism now has got a static example fallback in case the `dynamic` flag is not enabled #347
+* Prism can now validate servers #351
+
+### Chores
```
Comment (philsturgeon) [inline CHANGELOG.md]: Thank yiou but what I meant was that we dont need to list chores. Changelogs are for changes to the software, to let the user know what new features they are getting, what bugs were fixed, and what breaking changes happened. 

The users don't care if we cleaned our teeth and mowed the lawn during this time, just the prism changes.

Comment (chris-miaskowski) [reply CHANGELOG.md]: ditched

Comment (XVincentX) [reply CHANGELOG.md]: Ah ok fantastic, removing then

**Pull request merged**